deploy/olm-catalog/knative-eventing-operator/0.6.0/knative-eventing-operator.v0.6.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,199 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: knative-eventing-operator
+      deployments:
+      - name: knative-eventing-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: knative-eventing-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: knative-eventing-operator
+            spec:
+              containers:
+              - command:
+                - knative-eventing-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
This should be `metadata.annotations['olm.targetNamespaces']` and `gen-csv` will hopefully generate that for you correctly some day.

Reply:
@jcrossley3 ok, thanks for that pointer - I've fixed this